storybook/src/brand/assets/icons.docs.mdx

```diff
@@ -1,8 +1,9 @@
 {/* @license CC0-1.0 */}
 
 import { Meta } from "@storybook/blocks";
-import { AmsterdamIconGallery } from "../../docs/components/AmsterdamIconGallery";
+import { IconGallery } from "../../docs/components/IconGallery";
 import { StatusBadge } from "../../docs/components/StatusBadge";
```

Comment:
We can remove the beta warning here :tada: and at the Icon component page as well :tada: